recipes/flowtorch/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "flowtorch" %}
+{% set version = "0.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 775e69535dc54615a6c92d835e29fadc8d2bcc79ba13ce3eca06ee13ec9d02a5
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  {% set pyver = ">=3.7" %}
+  {% set torchver = ">=1.8.1" %}
```

Comment:
If the dependency versions are only mentioned once or twice, it's better to add them to the dependencies directly. This makes it easier for the reviewer and also saves some time rendering the recipe.

Reply:
Totally agree. I was testing a few things and so, using variables helped keep things consistent. Will hard code them in next commit.

Reply:
Done.